platforms/s390x/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM alpine:3.13 AS builder
```

Comment:
For now, we need to use the previous Dockerfile version on s390x, because we still need to build librdkafa, as alpine 3.14 has not made it to s390x.